test/end-to-end/test_tar_export

```diff
@@ -32,13 +32,13 @@ else
 fi
 
 # Remove tarball if already present
-[ -e *.tar.gz ] && rm *.tar.gz
+rm -f ./*.tar.gz
```

Comment:
See https://github.com/koalaman/shellcheck/wiki/SC2202#globs-dont-work-as-operands-in---use-a-loop